tests/distributions/test_distribution.py

```diff
@@ -166,253 +164,6 @@ def test_all_distributions_have_support_points():
-    def test_custom_dist_custom_support_point_multivariate(self, size):
-        def density_support_point(rv, size, mu):
-            return (pt.ones(size)[..., None] * mu).astype(rv.dtype)
-
-        mu_val = np.random.normal(loc=2, scale=1, size=5).astype(pytensor.config.floatX)
-        with Model():
-            mu = Normal("mu", size=5)
-            a = CustomDist(
-                "a",
-                mu,
-                support_point=density_support_point,
-                ndims_params=[1],
-                ndim_supp=1,
-                size=size,
-            )
-        assert isinstance(a.owner.op, CustomDistRV)
-        evaled_support_point = support_point(a).eval({mu: mu_val})
-        assert evaled_support_point.shape == (*to_tuple(size), 5)
-        assert np.all(evaled_support_point == mu_val)
-
-    @pytest.mark.xfail(
-        NotImplementedError,
-        reason="Support shape of multivariate CustomDist cannot be inferred. See https://github.com/pymc-devs/pytensor/pull/388",
-    )
-    @pytest.mark.parametrize(
-        "with_random, size",
-        [
-            (True, ()),
-            (True, (2,)),
-            (True, (3, 2)),
-            (False, ()),
-            (False, (2,)),
-        ],
-    )
-    def test_custom_dist_default_support_point_multivariate(self, with_random, size):
-        def _random(mu, rng=None, size=None):
-            return rng.normal(mu, scale=1, size=to_tuple(size) + mu.shape)
-
-        if with_random:
-            random = _random
-        else:
-            random = None
-
-        mu_val = np.random.normal(loc=2, scale=1, size=5).astype(pytensor.config.floatX)
-        with Model():
-            mu = Normal("mu", size=5)
-            a = CustomDist("a", mu, random=random, ndims_params=[1], ndim_supp=1, size=size)
-        assert isinstance(a.owner.op, CustomDistRV)
-        if with_random:
-            evaled_support_point = support_point(a).eval({mu: mu_val})
-            assert evaled_support_point.shape == (*to_tuple(size), 5)
-            assert np.all(evaled_support_point == 0)
-        else:
-            with pytest.raises(
-                TypeError,
-                match="Cannot safely infer the size of a multivariate random variable's support_point.",
-            ):
-                evaled_support_point = support_point(a).eval({mu: mu_val})
-
-    def test_dist(self):
-        mu = 1
-        x = pm.CustomDist.dist(
-            mu,
-            logp=lambda value, mu: pm.logp(pm.Normal.dist(mu), value),
-            random=lambda mu, rng=None, size=None: rng.normal(loc=mu, scale=1, size=size),
-            shape=(3,),
-        )
-
-        x = cloudpickle.loads(cloudpickle.dumps(x))
-
-        test_value = pm.draw(x, random_seed=1)
-        assert np.all(test_value == pm.draw(x, random_seed=1))
-
-        x_logp = pm.logp(x, test_value)
-        assert np.allclose(x_logp.eval(), st.norm(1).logpdf(test_value))
-
-
 class TestCustomSymbolicDist:
```

Comment:
These should also be moved

Reply:
Done in 34e41e5

I will update the test results in the PR description with the new figures:
```
33/0/33 (total)
38/14/52 (total)
```